From 8a8016f6d2af335ab205aa40d5274fc9b0c7a566 Mon Sep 17 00:00:00 2001
From: Steve James <ste@junkomatic.net>
Date: Sun, 5 Apr 2015 16:29:51 +0200
Subject: [PATCH 1/2] Fix compilation with g++ 4.8.2

Where db_iter.cc fails to get a typedef for ssize_t when compiled by
GCC.

Upstream-Status: Submitted [https://github.com/google/leveldb/issues/233]

Signed-off-by: Steve James <ste@junkomatic.net>
Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
---
 db/db_iter.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/db/db_iter.cc b/db/db_iter.cc
index 3b2035e..c2e5f35 100644
--- a/db/db_iter.cc
+++ b/db/db_iter.cc
@@ -13,6 +13,7 @@
 #include "util/logging.h"
 #include "util/mutexlock.h"
 #include "util/random.h"
+#include <sys/types.h> // for ssize_t
 
 namespace leveldb {
 
-- 
2.1.0

